cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -336,6 +336,18 @@ spec:
             {{- end }}
             {{- end }}
           env:
+            {{- if .Values.global.grafana}}
+            {{- if .Values.global.grafana.enabled }}
+            - name: GRAFANA_ENABLED
+              value: "true"
+            {{- else }}
+            - name: GRAFANA_ENABLED
+              value: "false"
+            {{- end }}
+            {{- else }}
+            - name: GRAFANA_ENABLED
+              value: "false"
+            {{- end }}
```

Comment:
I'd suggest doing something like:

https://github.com/kubecost/cost-analyzer-helm-chart/blob/64e022c33024a3944a568d6b29844033aca1e7f6/cost-analyzer/templates/cost-analyzer-deployment-template.yaml#L511-L513

```
{{- if .Values.global.grafana }}
- name: GRAFANA_ENABLED
   value: {{ (quote .Values.global.grafana.enabled) | default (quote false) }}
{{- end}}

And then in the code, when we check for the `env.IsGrafanaEnabled()` we should return `GetBool("GRAFANA_ENABLED", false)`

Reply:
Updated. Thanks for the tip! I'm not familiar with all the available things in templates like `quote` and `default`, but it definitely felt like this could be clearer.